sdk/core/Azure.Core.TestFramework/src/Instrumentation/j5gysqvq.jgm~

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
```

Comment:
Seems like a mistake?

Reply:
Good catch. It should be added by accident.